dll/shellext/netshell/lanstatusui.cpp

```diff
@@ -837,15 +842,24 @@ LANStatusDlg(
     switch (uMsg)
     {
         case WM_INITDIALOG:
-            pContext = (LANSTATUSUI_CONTEXT *)lParam;
-            SetWindowLongPtr(hwndDlg, DWLP_USER, (LONG_PTR)lParam);
-            pContext->nIDEvent = SetTimer(hwndDlg, 0xFABC, 1000, NULL);
             return TRUE;
         case WM_TIMER:
-            pContext = (LANSTATUSUI_CONTEXT*)GetWindowLongPtr(hwndDlg, DWLP_USER);
-            if (wParam == (WPARAM)pContext->nIDEvent)
+            pContext = NULL;
+            if(!pContext)
```

Comment:
The test will always be TRUE.

Reply:
I simplified my initial design but forgot this part. Fixed in last commit.